README.md

```diff
@@ -27,81 +15,88 @@ This repo contains a few critical pieces:
 	1. An ontology validator
 	2. A loadsheet validator
 	3. A handler class that sits atop all the relevant classes
-	4. A rules engine for applying regular expression pattern matching.
+	4. A rules engine for applying regular expression pattern matching
 	5. A representations class set for converting the loadsheet into ontology-usable objects
 
+### Dependencies
+This repo requires a few libraries be installed prior to use:
+1. pyyaml (for parsing YAML documents)
+2. pyfiglet (for fancy CLI name)
+3. openpyxl (for Excel read/write)
+4. pandas (for loadsheet backend)
+5. typing (for type checking)

```

Comment:
typing is part of the standard python library so doesn't need to be listed as a dependency.